geopandas/base.py

```diff
@@ -328,6 +328,47 @@ def is_empty(self):
         """
         return _delegate_property("is_empty", self)
 
+    @property
+    def count_coordinates(self):
+        """
+        Returns a ``Series`` containing the count of the number of coordinate pairs
+        in a geometry array.
+
+        Examples
+        --------
+        An example of a GeoDataFrame with two line strings, one point and one None
+        value:
+        >>> from shapely.geometry import LineString, Point
```

Comment:
```suggestion

        >>> from shapely.geometry import LineString, Point
```

There needs to be an empty line to ensure proper formatting. Check https://geopandas--3026.org.readthedocs.build/en/3026/docs/reference/api/geopandas.GeoSeries.count_coordinates.html